### DISCUSSION TITLE
WebView2 - google sign in is refused

### DISCUSSION THREAD

**Discussion by JohnDizzle** [microsoft/WindowsAppSDK#1810] [Q&A] [answered]
I am using the windowsappsdk 1.0.0-expermential1 for uap.  I have built a browser and when I try to sign into google it's refusing sign-in.  I look at my google account a selected un-secured app access to true/on.   I still get refused because the browser isn't supported.  ?

**Comment by andrewleader** [comment, marked as answer]
Hi @JohnDizzle, since this is a WebView2 question, I recommend asking your question on the WebView2 repo: https://github.com/MicrosoftEdge/WebView2Feedback/issues
